### DISCUSSION TITLE
Doorbell button integration

### DISCUSSION THREAD

**Discussion by dnestico** [mrlt8/docker-wyze-bridge#300] [Ideas]
This plugin is awesome, love how easy it to setup and it just works!

1 question I have though is it has my doorbell live view, so is it possible in the future to integrate the button on the doorbell?

I can use that button when pushed for automation and it would be awesome. Thanks.

**Comment by mrlt8** [comment]
I don't have a wyze doorbell, so not exactly sure how the push buttons work, but I believe they might have been using SIP to initiate a "call" to the app..? I believe there are some SIP plugins for home assistant/homebridge, so that may be an option?

**Reply by dnestico** [reply]
No idea because I am not that savvy when it comes to this stuff haha, but it's using VOIP on my phone to make a phone call whenever the button is pushed which I enabled VOIP, I will do more research on this "SIP" thing later but maybe now that you know it has VOIP call integration that might be of more help to you?

**Reply by cheme75** [reply]
Note that the db press also triggers the chime accessory so there must be a momentary toggle of some variable or flag in the fw.

**Reply by dnestico** [reply]
Very true 👀

I'm currently testing an automaton that uses Alexa to tell HA my doorbell was triggered, just waiting for someone to actually push it lol..

But if you figure out something better then let us know!

**Reply by mrlt8** [reply]
> Note that the db press also triggers the chime accessory so there must be a momentary toggle of some variable or flag in the fw.

There is some "DDQ" related stuff in the app that seems to be related to the doorbell: `K11802_getDDQAlarmPeriod` and `K11800_getDDQVolumeTsmes`

**Reply by cheme75** [reply]
Beyond my tech level! I haven’t looked inside any app since I did a jailbreak on my original iPhone 3GS and read how to edit some files to customize certain things. Just thought I’d point out some aspect of functionality that might lend itself to external usefulness.

**Comment by dnestico** [comment]
So my doorbell was finally fun today and can confirm this automation works!!!

Now just need the camera to not be delayed because the snapshot it showed was from hours ago 😬
